docs/src/styles/docs/base.scss

```diff
@@ -196,7 +193,13 @@ h3:hover > a .icon-link {
 
   --amplify-components-heading-1-font-size: 3rem;
   --amplify-components-heading-1-font-weight: 750;
-  
+
   /* custom vars */
   --docs-header-height: 4rem;
 }
+
+.container {
```

Comment:
This is the only non-prettier change in this file.

Reply:
Dang did prettier not run on my commit to this file?